src/main/common/chirp.c

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Incorrect license header. Please see default license in /

Reply:
Check, done.